rosetta/failure/invalid_proposal_key.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2021 Optakt Labs OÜ
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+package failure
+
+import (
+	"fmt"
+
+	"github.com/onflow/flow-go/model/flow"
+)
+
+type InvalidProposalKey struct {
+	Description Description
+	Address     flow.Address
+	Index       int
+}
+
+func (i InvalidProposalKey) Error() string {
+	return fmt.Sprintf("invalid proposal key (address: %s, key index: %d): %s", i.Address.String(), i.Index, i.Description)
```

Comment:
You know that `%s` automatically uses the `String()` function, right? :D

Reply:
That used to be a `%v`, but I changed it to be explicit, forgot that one :D